### ISSUE TITLE
Adding a new project Fashion MNIST dataset Image Classification 

### ISSUE THREAD

**Issue by deedGhost** [abhisheks008/DL-Simplified#573]
### Deep Learning Simplified Repository (Proposing new issue)
:red_circle: **Project Title** : </br> Fashion MNIST Image Classification
:red_circle: **Aim** : </br>This project focuses on image classification task using deep learning model on the Fashion MNIST dataset. 
:red_circle: **Dataset** : </br>https://www.kaggle.com/datasets/zalando-research/fashionmnist
:red_circle: **Approach** : I will use a CNN model to perform the task of image classification. I will use Early Stopping in the developed model. Additionally, a GUI using Tkinter to provide a user-friendly way to interact with the model.

**********************************************************************************
### 📍 **Follow the Guidelines to Contribute in the Project :**
* You need to create a separate folder named as the Project Title.
* Inside that folder, there will be four main components.
   * Images - To store the required images.
   * Dataset - To store the dataset or, information/source about the dataset.
   * Model - To store the machine learning model you've created using the dataset.
   * `requirements.txt` - This file will contain the required packages/libraries to run the project in other machines.
* Inside the `Model` folder, the `README.md` file must be filled up properly, with proper visualizations and conclusions.

***********************************************************************
:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
- "Issue Title" and "PR Title should be the same. Include issue number along with it.
- Follow Contributing Guidelines & Code of Conduct before start Contributing.

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : Anjali
- GitHub Profile Link : https://github.com/deedGhost
- Email ID : anjali17356@gmail.com
- Participant ID (if applicable): Anjali (discord: .sleepyded)
- Approach for this Project : Deep learning model for Image Classification.
- What is your participant role? (Mention the Open Source program)
GSSOC'24 Contributor
*************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by VoonnaHarsha** [comment]
@deedGhost  , im looking forward to contribute Fashion MNIST Image Classification
can yu assign me task 

discord name : harsha00429
mail: harshavoonna@gmail.com
github: https://github.com/VoonnaHarsha

**Comment by abhisheks008** [comment]
Hi @deedGhost can you share a brief on your approach? What are the architectures/methods you are planning for this project?

Hi @VoonnaHarsha as this issue is raised by @deedGhost, hence this can be assigned to only Anjali. You can check out other issues, which are open and opened by me (@abhisheks008)

**Comment by deedGhost** [comment]
@abhisheks008 I will be using a CNN model as CNNs are widely used for image classification tasks. These architectures will have convolutional layers, pooling layers, and fully connected layers. I will be incorporating early stopping method to avoid the overfitting problem and then will compare it with the normal model. Additionally, I am planning to create an interactive GUI for the working of this model.

**Comment by abhisheks008** [comment]
Okay! So you are planning to create a web app for the same. Then please follow the project structure as, 
```
Project Dummy
|- Dataset
  |- dataset.csv
  |- README.md
|- Images
  |- img1.png
  |- img2.png
  |- img3.png
|- Model
  |- project_dummy.ipynb
  |- README.md
|- Web App
  |- app.py
  |- templates
  |- demo.mp4
  |- README.md
|- requirements.txt
```

For your reference you can check out these projects which are already in this repo, [Brain Tumor Detection](https://github.com/abhisheks008/DL-Simplified/tree/main/Brain%20Tumor%20Detection), [Alzheimers Detection](https://github.com/abhisheks008/DL-Simplified/tree/main/Alzheimers%20Detection).

Check out the Web App README template [here](https://github.com/abhisheks008/DL-Simplified/blob/main/.github/web-app_readme_template.md).

Check out the project README template [here](https://github.com/abhisheks008/DL-Simplified/blob/main/.github/readme_template.md).

@deedGhost looking forward to hearing from you.

**Comment by deedGhost** [comment]
@abhisheks008 Okay sir! Thank you for the related materials. I will start working on it.

**Comment by abhisheks008** [comment]
Assigned to you @deedGhost

**Comment by deedGhost** [comment]
Hey @abhisheks008
I want to ask which images do I need to add in the Images folder, the ones I used to test the model or something else??

**Comment by abhisheks008** [comment]
> Hey @abhisheks008 I want to ask which images do I need to add in the Images folder, the ones I used to test the model or something else??

The EDA results, accuracy graphs, confusion matrix so on. All the resultant images that you have generated while developing the project. Also these EDA results will gonna be a part of the README too.

**Comment by Akshitha2912v** [comment]
Hi,I would like to work on this issue under GSSoC 2026.
Could you please assign this issue to me?
Thank you!

**Comment by Pritosh2387** [comment]
Hi @abhisheks008,

I'm Pritosh Burman, a GSSoC 2026 Contributor, and I'd like to work on this issue.

**Full Name:** Pritosh Burman
**GitHub:** https://github.com/Pritosh2387
**Email:** pritoshburman2387@gmail.com
**Participant Role:** GSSoC 2026 Contributor

### Proposed Approach
I plan to implement a CNN-based Fashion MNIST image classification model using TensorFlow/Keras with:
- Data preprocessing
- CNN architecture
- Early stopping
- Accuracy/loss graphs
- Confusion matrix
- Tkinter GUI
- Complete documentation following the repository guidelines.

Please assign this issue to me.

Thank you!
